resmio_conversion.js

```diff
@@ -0,0 +1,195 @@
+/*!
+ * ===========================================
+ * General Conversion Tracking v1.0
+ * Copyright 2017 resmio GmbH (Philipp Sahner)
+ * ===========================================
+ */
++function ($) {
+  /*
+   * +++++++++++++++++++++++++++++++++++++++++++++++++++++
+   * Check the url for parameters when document has loaded
+   * and handle it accordingly
+   * +++++++++++++++++++++++++++++++++++++++++++++++++++++
+   */
+  $(document).ready(function(){
+    // get the url, check it for parameters and update the links
+ 	  var url = window.location.href,
+        obj = getAllUrlParams(url);
+        isGclid = false;
+        isObj = false;
+    if (!($.isEmptyObject(obj))) {
+      isObj = true;
+      // create the cookies for the parameters
+      $.each( obj, function( key, value ) {
+        var current_value = readCookie(key),
+            value = value.toString().toLowerCase();
+        if (value != current_value) {
+          createCookie(key, value, 90);
+        }
+        if (key == 'gclid') {
+          isGclid = true;
```

Comment:
If you set i`sGclid` here to true and later in the loop you have something with a different key you will set `isGclid` back to false. 

A good rule of thumb is to NEVER change the value of a external variable in a loop ;)